pkg/apis/autoscaling/register.go

```diff
@@ -43,13 +43,13 @@ const (
 
 	// MinScaleAnnotationKey is the annotation to specify the minimum number of Pods
 	// the PodAutoscaler should provision. For example,
-	//   autoscaling.knative.dev/min-scale: "1"
-	MinScaleAnnotationKey = GroupName + "/min-scale"
```

Comment:
You need to update `MaxScaleAnnotation` and `MinScaleAnnotation` to preserve the old behaviour

Reply:
@dprotaso I have just renamed the variable here to explicitly mention that `/min-scale` is getting deprecated. See the line below. I am preserving the `/min-scale` annotation for now.